AGENTS.md

```
@@ -2,26 +2,34 @@
 ## Build/Test Commands
 - `npm run dev` - Start development server
-- `npm run build` - Build for production
+- `npm run build` - Build for production  
 - `npm run preview` - Preview production build
 - No test framework configured - add tests if needed
+- No linting configured - use built-in TypeScript checking
 
 ## Project Structure
-- Astro v5 project with TypeScript
-- Uses strict TypeScript config (`astro/tsconfigs/strict`)
+- Astro v5 project with TypeScript (strict mode)
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)